src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -339,6 +344,8 @@ def get(self, request: Request, organization: Organization | RpcOrganization) ->
             elif threshold_type == ReleaseThresholdType.NEW_ISSUE_COUNT:
                 metrics.incr("release.threshold_health_status.check.new_issue_count")
```

Comment:
Ok!

So - since our new issue counts are coming simply from a model, we actually _can_ construct a bulk query!

Following the error flow above,
Could we pull together a helper fetch that pulls all the `first_seen` issues where the project_ids are `__in` the project_id_list, release_values_list (or release_id_list), etc. etc. ?

Then we can construct a separate health check `is_new_issue_count_healthy` which contains the logic to parse through the bulk data and determine whether an individual threshold is healthy or not?